gator/hub/__main__.py

```diff
@@ -26,12 +27,16 @@
 @click.option(
     "--static",
     type=click.Path(exists=True, file_okay=False, path_type=Path),
-    default=Path(__file__).parent.parent.parent / "gator-hub",
+    default=Path(
+        os.environ.get(
+            "GATOR_HUB_ROOT", Path(__file__).parent.parent.parent / "gator-hub"
+        )
+    ),
 )
 # Postgres DB
 @click.option("--db-host", default="127.0.0.1", type=str)
 @click.option("--db-port", default=5432, type=int)
-@click.option("--db-name", default="gator", type=str)
+@click.option("--db-name", default="postgres", type=str)
```

Comment:
I couldn't work out how to modify the db name from the postgres gui so changed this to use the default instead